### PR TITLE
Update the event on Title

### PR DESCRIPTION
**Pull request by jiachiwen**
Help to Update the event on Title

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # hacktoberfest2021-markdown-playground
-Markdown playground for everyone. Enjoy [hacktoberfest](https://hacktoberfest.digitalocean.com/) 2021!
+Markdown playground for everyone. Enjoy [hacktoberfest2021](https://hacktoberfest.digitalocean.com/) 2021!
```
Comment (hms5232) [inline README.md]: 這邊應該不用喔，我後面有 `2021` 了，而且加上年分只是為了區隔是哪年舉辦的，活動基本上還是稱 `hacktoberfest` 就好。
```suggestion
Markdown playground for everyone. Enjoy [hacktoberfest](https://hacktoberfest.digitalocean.com/) 2021!
```

**Review by hms5232 — COMMENTED** [review]
How about my suggestion?

**Comment by jiachiwen** [comment]
Update about the PR in Title

File: README.md (hunk 1 above)
Comment (jiachiwen) [reply README.md]: Sure, let me see

**Comment by hms5232** [comment]
I'm not sure what happen. I will check how this occurs.

**Comment by hms5232** [comment]
After I clone both two repo and diff, it doesn't exist any difference! What happen to this 🤔

**Review by hms5232 — APPROVED** [review]
OK, I think that some problem occur on Github, or some magic happen there.

I will merge this PR as a experimentation. If git work correctly, nothing would be changed.

**Comment by hms5232** [comment]
Good! Nothing happen. So, what's going wrong in files changed tab? 😵

**Pull request merged**